bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -0,0 +1,449 @@
+{#
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol.html" %}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
+
+{% block page_title %}{{ ftl('firefox-home-firefox-protect-your') }}{% endblock %}
+
+{% block page_title_suffix %}{{ ftl('firefox-home-mozilla') }}{% endblock %}
+
+{% block page_desc %}{{ ftl('firefox-home-firefox-is-more-than') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+{% block page_favicon %}{{ static('img/favicons/firefox/favicon.ico') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/favicons/firefox/favicon-196x196.png') }}{% endblock %}
+{% block page_ios_icon %}{{ static('img/favicons/firefox/apple-touch-icon.png') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-master') }}
```

Comment:
Question: does this page also rely on the styles from the main `/firefox` page? They don't seem to (which is probably a good thing), so can you remove this stylesheet?